src/compiler/transformers/test/map-imports-to-path-aliases.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { mockValidatedConfig } from '@stencil/core/testing';
+import { ValidatedConfig } from '../../../internal';
+import { transpileModule } from './transpile';
+import ts, { Extension } from 'typescript';
+import { mapImportsToPathAliases } from '../map-imports-to-path-aliases';
+
+describe('mapImportsToPathAliases', () => {
+  let module: ReturnType<typeof transpileModule>;
+  let config: ValidatedConfig;
+  let resolveModuleNameSpy: jest.SpyInstance<
+    ReturnType<typeof ts.resolveModuleName>,
+    Parameters<typeof ts.resolveModuleName>
+  >;
+
+  beforeEach(() => {
+    config = mockValidatedConfig({ tsconfig: './tsconfig.json', tsCompilerOptions: {} });
+
+    resolveModuleNameSpy = jest.spyOn(ts, 'resolveModuleName');
+  });
+
+  afterEach(() => {
+    resolveModuleNameSpy.mockReset();
+  });
+
+  it('should ignore relative imports', () => {
```

Comment:
Consider making these test descriptions assertive to minimize repetition of 'should':
```suggestion
  it('ignores relative imports', () => {
```